tests_tf/test_attacks.py

```diff
@@ -1030,23 +1031,45 @@ def test_generate_np_gives_clipped_adversarial_examples(self):
         self.assertTrue(np.max(x_adv) < .301)
 
 
+class SimpleSpatialBrightPixelModel(Model):
+    """
+    If there is a bright pixel in the image returns the first class.
+    Otherwise returns the second class. Spatial attack should push the
+    bright pixels off of the image
+    """
+
+    def __init__(self, scope='simple', nb_classes=2, **kwargs):
+        del kwargs
+        Model.__init__(self, scope, nb_classes, locals())
+
+    def fprop(self, x, **kwargs):
+        del kwargs
+
+        flat_x = slim.flatten(x)
+        first_logit = tf.reduce_max(flat_x, axis=1)
+        second_logit = tf.ones_like(first_logit) * 0.5
+        res = tf.stack([second_logit, first_logit], axis=1)
+        return {self.O_LOGITS: res,
+                self.O_PROBS: tf.nn.softmax(res)}
+
+
 class TestSpatialTransformationMethod(CleverHansTest):
     def setUp(self):
         super(TestSpatialTransformationMethod, self).setUp()
 
         self.sess = tf.Session()
-        self.model = DummyModel(scope='dummy_model_spatial')
+        self.model = SimpleSpatialBrightPixelModel(scope='dummy_model_spatial')
```

Comment:
I guess a more special scope name like "bright_pixel_model" since this is not using DummyModel anymore.